cmd/kubebuilder/initproject/init.go

```diff
@@ -43,13 +45,21 @@ var domain string
 var copyright string
 var bazel bool
 var controllerOnly bool
+var newVersion bool
 
 func AddInit(cmd *cobra.Command) {
 	cmd.AddCommand(repoCmd)
 	repoCmd.Flags().StringVar(&domain, "domain", "", "domain for the API groups")
 	repoCmd.Flags().StringVar(&copyright, "copyright", filepath.Join("hack", "boilerplate.go.txt"), "Location of copyright boilerplate file.")
 	repoCmd.Flags().BoolVar(&bazel, "bazel", false, "if true, setup Bazel workspace artifacts")
 	repoCmd.Flags().BoolVar(&controllerOnly, "controller-only", false, "if true, setup controller only")
+	repoCmd.Flags().BoolVar(&newVersion, "newkb", false, "if true, init project with kubebuilder 1.0")
+
+	prj = ProjectForFlags(repoCmd.Flags())
+	bp = BoilerplateForFlags(repoCmd.Flags())
+	gopkg = &project.GopkgToml{}
+	mrg = &manager.Cmd{}
+	dkr = &manager.Dockerfile{}
```

Comment:
I am not able to see the scope of the above variables, but would highly discourage the use of globals.

Reply:
I put them inside a `projectOptions` struct